### PR TITLE
Adds prescription safety goggles/glasses

### PR DESCRIPTION
**Pull request by SapphicOverload**
Adds prescription safety goggles and glasses

### CODE REVIEW

**Comment by SapphicOverload** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -223,6 +223,11 @@ BLIND     // can't see anything
 	item_flags = AIRTIGHT|THICKMATERIAL
 	unacidable = 1
 
+/obj/item/clothing/glasses/safety
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:226]: path.

File: code/modules/clothing/glasses/glasses.dm [hunk 2]
```diff
@@ -223,6 +223,11 @@ BLIND     // can't see anything
...
+	name = "prescription safety glasses"
+	desc = "A simple pair of safety glasses. Thinner than their goggle counterparts, and comes with prescription overlay."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:228]:
```suggestion
	desc = "A simple pair of safety glasses. Thinner than their goggle counterparts and comes with a prescription overlay."
```

File: code/modules/clothing/glasses/glasses.dm [hunk 3]
```diff
@@ -260,6 +265,11 @@ BLIND     // can't see anything
 	update_icon()
 	usr.update_action_buttons()
 
+/obj/item/clothing/glasses/safety/goggles/prescription
+	name = "prescription safety goggles"
+	desc = "A simple pair of safety goggles. It's general chemistry all over again. Comes with prescription overlay."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:270]:
```suggestion
	desc = "A simple pair of safety goggles. It's general chemistry all over again. Comes with a prescription overlay."
```

File: code/modules/client/preference_setup/loadout/loadout_eyes.dm [hunk 4]
```diff
@@ -66,6 +67,7 @@
 	..()
 	var/list/goggles = list()
 	goggles["goggles, safety"] = /obj/item/clothing/glasses/safety/goggles
+	goggles["goggles, safety, prescription"] = /obj/item/clothing/glasses/safety/goggles/prescription
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_eyes.dm:70]:
```suggestion
	goggles["goggles, safety (prescription)"] = /obj/item/clothing/glasses/safety/goggles/prescription
```

File: code/modules/client/preference_setup/loadout/loadout_eyes.dm [hunk 5]
```diff
@@ -24,6 +24,7 @@
 	glasses["glasses, jamjar"] = /obj/item/clothing/glasses/regular/jamjar
 	glasses["glasses, monocle"] = /obj/item/clothing/glasses/monocle
 	glasses["glasses, safety"] = /obj/item/clothing/glasses/safety
+	glasses["glasses, safety, prescription"] = /obj/item/clothing/glasses/safety/prescription
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_eyes.dm:27]:
```suggestion
	glasses["glasses, safety (prescription)"] = /obj/item/clothing/glasses/safety/prescription
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
you need a changelog as well.

**Review by SleepyGemmy — APPROVED** [review]

**Review by KingOfThePing — APPROVED** [review]

**Pull request merged**